cmd/appgw-ingress/main.go

```diff
@@ -134,7 +134,9 @@ func main() {
 		azClient.SetAuthorizer(authorizer)
 	}
 
-	if _, err = azClient.GetGateway(); err != nil {
+	// check if application gateway exists/have get access
+	err = azClient.WaitForGetAccessOnGateway()
```

Comment:
Can we talk about this offline. What are we waiting for contributor access after wards ? Also WaitForGetAccessOnGateway prints that we are waiting for contributor access too ? Is that a typo ?

Reply:
sure

Reply:
No, it is not a typo. I added it to this message as well as so that user can see the required permission all at once.